pages/services/edge-lb/1.0.0/pool-configuration/index.md

```diff
@@ -0,0 +1,504 @@
+---
+layout: layout.pug
```

Comment:
@drewkerrigan could we have to different `pool-configuration` docs. One for `v1` and one for `v2`. Putting both together here is going to confuse the users, especially the ones coming infrom v 0.1.9?